pkg/controller/nodelifecycle/node_lifecycle_controller_test.go

```diff
@@ -1829,6 +1861,583 @@ func TestMonitorNodeHealthUpdateStatus(t *testing.T) {
 	}
 }
 
+func TestMonitorNodeHealthUpdateNodeAndPodStatusWithLease(t *testing.T) {
```

Comment:
Why "AndPod" ?

Reply:
Because this test checks both node and pod status updates.